R/derive_vars_transposed.R

```diff
@@ -96,6 +97,20 @@
 #'     value_var = FASTRESC
 #'   ) %>%
 #'   select(USUBJID, CMDECOD, starts_with("CMATC"))
+#'
+#' # Note: the id_vars argument here is key to uniquely identify
+#' # rows of dataset_merge and avoid duplicates-related errors.
+#' # Compare the above call with:
+#'
+#' # cm %>%
+#' #   derive_vars_transposed(
+#' #     facm,
+#' #     by_vars = exprs(USUBJID, CMREFID = FAREFID),
+#' #     id_vars = NULL,
+#' #     key_var = FATESTCD,
+#' #    value_var = FASTRESC
+#' #  )
```

Comment:
Please use `try()` instead of commenting out the code. Then the error message is displayed in the documentation.

Reply:
Thanks - done.